common/src/main/java/org/apache/syncope/common/services/ReportService.java

```diff
@@ -192,4 +195,25 @@
     @Consumes({ MediaType.APPLICATION_XML, MediaType.APPLICATION_JSON })
     Response exportExecutionResult(@NotNull @PathParam("executionId") Long executionId,
             @QueryParam("format") ReportExecExportFormat fmt);
+    
+    
+    /**
+     * List all the reports of a particular type
+     *
+     * @param type of report (like All, Running, Scheduled)
```

Comment:
Same here: "type of report job" (no need to give examples, you have a proper Enum)